packages/create-llama/helpers/python.ts

```diff
@@ -154,28 +180,7 @@ export const installPythonTemplate = async ({
   const addOnDependencies = getAdditionalDependencies(vectorDb);
   await addDependencies(root, addOnDependencies);
 
-  // install python dependencies
-  if (installDependencies) {
-    if (isPoetryAvailable()) {
-      console.log(
-        `Installing python dependencies using poetry. This may take a while...`,
-      );
-      const installSuccessful = tryPoetryInstall();
-      if (!installSuccessful) {
-        console.warn(
-          yellow("Install failed. Please install dependencies manually."),
-        );
-      }
-    } else {
-      console.warn(
-        yellow(
-          `Poetry is not available in the current environment. The Python dependencies will not be installed automatically.
-Please check ${terminalLink(
-            "Poetry Installation",
-            `https://python-poetry.org/docs/#installation`,
-          )} to install poetry first, then install the dependencies manually.`,
-        ),
-      );
-    }
+  if (postInstallAction) {
```

Comment:
is that condition correct? looks like it would also be executed for `none`

Reply:
ah, my bad. this one is not covered by in tests